K8s-deployment/K8s-cluster/addons/cluster-autoscaler/azure/cluster-autoscaler-autodiscover.yaml

```diff
@@ -168,17 +168,19 @@ spec:
           name: cluster-autoscaler
           resources:
             limits:
-              cpu: 100m
-              memory: 300Mi
+              cpu: 150m
+              memory: 512Mi
             requests:
```

Comment:
You might have interchanged the limits with requests key, limits values should be greater to requests